jme3-examples/gradle.properties

```diff
@@ -2,4 +2,4 @@
 assertions = true
 
 # Build javadoc per Github issue #1366
-buildJavaDoc = true
\ No newline at end of file
+buildJavaDoc = true

```

Comment:
You can also restore this file, as it doesn’t contribute to the functionality of your PR